### ISSUE TITLE
 Error: Input Object type NullableJsonFilter must define one or more fields.

### ISSUE THREAD

**Issue by nikolasburk** [graphql-nexus/nexus-plugin-prisma#666]
I have this Prisma schema:

```prisma
model User {
  id       String   @id
  email    String?  @unique
  name     String
  role     Role?    @default(CUSTOMER)
  jsonData Json?
  posts    Post[]
}

model Post {
  id         String     @id
  createdAt  DateTime   @default(now())
  updatedAt  DateTime   @updatedAt
  content    String?
  published  Boolean    @default(false)
  title      String
  author     User?      @relation(fields: [authorId], references: [id])
  authorId   String?
}

enum Role {
  ADMIN
  CUSTOMER

  @@map("User_role")
}
```

When trying to project relations on my model with `filtering` and `ordering` set to `true`:

```ts
 schema.objectType({
  name: 'User',
  definition(t) {
    t.model.id()
    t.model.email()
    t.model.name()
    t.model.posts({
      filtering: true,
      ordering: true,
    })
  },
})
schema.objectType({
  name: 'Post',
  definition(t) {
    t.model.id()
    t.model.title()
    t.model.content()
    t.model.published()
  },
})
```

I get this error:

```
2134 ● nexus:dev:watcher restarting  --  type: 'change'  file: 'src/generated/nexus.graphql'
 560 ● server listening  --  url: 'http://localhost:4000/'
4294 ✕ app unhandledRejection
       | error  Error: Input Object type NullableJsonFilter must define one or more fields.
       |            at assertValidSchema (/Users/nikolasburk/Desktop/upgrade-guide/prisma-binding/hello-world-mysql/node_modules/graphql/type/validate.js:71:11)
       |            at Object.validate (/Users/nikolasburk/Desktop/upgrade-guide/prisma-binding/hello-world-mysql/node_modules/graphql/validation/validate.js:54:35)
       |            at /Users/nikolasburk/Desktop/upgrade-guide/prisma-binding/hello-world-mysql/node_modules/nexus/dist/runtime/server/handler-graphql.js:28:42
       |            at processTicksAndRejections (internal/process/task_queues.js:89:5)
       |            at async /Users/nikolasburk/Desktop/upgrade-guide/prisma-binding/hello-world-mysql/node_modules/nexus/dist/runtime/server/server.js:87:9
```

Note that everything seems to work fine if I don't include `filtering` and ordering:

```ts
// this works
 schema.objectType({
  name: 'User',
  definition(t) {
    t.model.id()
    t.model.email()
    t.model.name()
    t.model.posts()
  },
})
schema.objectType({
  name: 'Post',
  definition(t) {
    t.model.id()
    t.model.title()
    t.model.content()
    t.model.published()
  },
})
```

![image](https://user-images.githubusercontent.com/4058327/81962931-b9446100-9614-11ea-8e58-1015430dc824.png)

**Comment by nikolasburk** [comment]
Ok, it seems like the issue came from the `Json` field on `User` in my Prisma schema. Commenting that you resolves the issue.

**Comment by joshchuable** [comment]
I also have this issue, but when I try to use updateOneModel in the nexus mutations. @nikolasburk, were you able to get around this without forgoing the JSON field?

**Comment by allesklarbeidir** [comment]
I tried the following and it seems to work:

I added 
```
extendInputType({
    type: "JsonFilter",
    definition(t){
        t.string("DUMMY_INPUT_FIELD_WORKAROUND");
    }
}),
extendInputType({
    type: "NullableJsonFilter",
    definition(t){
        t.string("DUMMY_INPUT_FIELD_WORKAROUND");
    }
}),
```

And now there's no error and GraphQL Playground shows up as normal.

**Comment by Weakky** [comment]
Closed by #690

**Comment by johhansantana** [comment]
> Closed by #690

how do we upgrade to this?
